src/snowcli/cli/nativeapp/commands.py

```diff
@@ -135,7 +135,8 @@ def app_teardown(
     **options,
 ) -> CommandResult:
     """
-    Drops an application and an application package as defined in the project definition file.
+    Attempts to drop both the application and package as defined in the project definition file.
```

Comment:
nit: any reason we change application package to package terminology?

Reply:
No particular reason outside of the fact that there's nothing else package could refer to, so I often shorten it.